modules/dynalite/src/main/java/org/testcontainers/dynamodb/DynaliteContainer.java

```diff
@@ -0,0 +1,76 @@
+package org.testcontainers.dynamodb;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import org.junit.rules.ExternalResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+
+/**
+ * Container for Dynalite, a DynamoDB clone.
+ */
+public class DynaliteContainer extends ExternalResource {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DynaliteContainer.class);
+    private final GenericContainer delegate;
+
+    public DynaliteContainer() {
+        this("richnorth/dynalite:v1.2.1-1");
```

Comment:
Maybe it's time to move this image to quay.io/testcontainers?

Reply:
Good idea...